transformer_engine/pytorch/fp8.py

```diff
@@ -538,12 +538,9 @@ def _default_sf_compute(
     margin: int,
 ) -> torch.Tensor:
     """Default function to convert amax to scaling factor."""
-    exp = torch.floor(torch.log2(fp8_max / amax)) - margin
```

Comment:
Hi @ksivaman, thanks for making this change! I wonder whether this change requires other kernel or code changes? The reason why I am asking is that [this doc](https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/common.html) mentions "The scaling factor should always be a power of 2 to not introduce numerical error during the conversion from FP8 to higher precision format."